### ISSUE TITLE
Положение курсора в окне "Go to page"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#45]
```
Honeycomb 3.1

Курсор в изначальном состоянии находится 
перед цифрой. приходится перемещать его 
пальцем а потом затирать через BackSpace. 
Предлагаю, чтобы при появлении окна цифры 
были уже выделенными, тогда можно будет 
сразу вбивать номер страницы.
```

Original issue reported on code.google.com by `imr...@gmail.com` on 3 Sep 2011 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
+1. Будет действительно удобно. Ну или хотя 
бы чтоб курсор по умолчанию всегда вконец 
становился.
```

Original comment by `Original...@gmail.com` on 3 Sep 2011 at 11:54

**Comment by GoogleCodeExporter** [comment]
```
Если это можно сделать без сильного 
влезания в потроха контрола - сделаем.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 3 Sep 2011 at 6:20
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Решил не создавать отдельное issue и написать 
сюда (так как тема практически идентична).

Неплохо бы тоже самое реализовать и для 
закладок, когда их создаёшь.
```

Original comment by `Original...@gmail.com` on 5 Sep 2011 at 7:57

**Comment by GoogleCodeExporter** [comment]
```
Implemented in trunk. Will be available since 1.2-rc5
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Sep 2011 at 8:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Спасибо, за оперативную реакцию!

P.S.: Прошу прощения за назойливость, что вы 
можете сказать по этому поводу: 
http://code.google.com/p/ebookdroid/issues/detail?id=19#c6 ?
```

Original comment by `imr...@gmail.com` on 5 Sep 2011 at 9:32
